symfony/src/Entity/Order.php

```diff
@@ -644,14 +671,44 @@ public function setFinalizedSite(?string $finalizedSite): self
         return $this;
     }
 
+    public function getSamples()
+    {
+        return $this->samples;
+    }
+
+    public function getSamplesInformation()
+    {
+        return $this->samplesInformation;
+    }
+
+    public function getSalivaSamples()
+    {
+        return $this->salivaSamples;
+    }
+
+    public function getSalivaSamplesInformation()
+    {
+        return $this->salivaSamplesInformation;
+    }
+
+    public function getSalivaInstructions()
+    {
+        return $this->salivaInstructions;
+    }
+
+    public function getCurrentVersion()
+    {
+        return $this->currentVersion;
+    }
+
     public function loadSamplesSchema($params = [])
     {
-        $currentVersion = $this->getVersion();
+        $this->currentVersion = $this->getVersion();
         if (!empty($params['order_samples_version'])) {
-            $currentVersion = $params['order_samples_version'];
+            $this->currentVersion = $params['order_samples_version'];
```

Comment:
Shouldn't the entity's version, if it exists, take precedence over the configured current version? That is, if you pull up an old order with a different version from the current version, we would want to load the sample information for the old version.